### PR TITLE
Update storage_engine.md

### PR DESCRIPTION
**Pull request by beckettsean**

### CODE REVIEW

**Comment by beckettsean** [comment]
@jwilder @pauldix I've polished the Storage Engine page and tried to make it a reference first and a story second. 

Please check my work for lies or omissions, thanks!

File: content/influxdb/v1.0/concepts/storage_engine.md [hunk 1]
```diff
@@ -7,226 +7,77 @@ menu:
...
+### The InfluxDB Storage Engine and the Time-Structured Merge Tree (TSM) 
 
 The new InfluxDB storage engine looks very similar to a LSM Tree.
 It has a write ahead log and a collection of read-only data files which are  similar in concept to SSTables in an LSM Tree.
-These data files, TSM files, contain sorted, compressed series data.
+TSM files contain sorted, compressed series data.
 
-InfluxDB will create a shard for each block of time.
-For example, if you have a retention policy with an unlimited duration, shards will get created for each 7 day block of time.
+InfluxDB will create a [shard]((/influxdb/v1.0/concepts/glossary/#shard) for each block of time.
```
Comment (jwilder) [inline content/influxdb/v1.0/concepts/storage_engine.md]: Extra parens `(` in this link at the beginning.

File: content/influxdb/v1.0/concepts/storage_engine.md [hunk 2]
```diff
@@ -331,86 +179,258 @@ When timestamps have this structure, they are scaled by the largest common divis
 This has the effect of converting very large integer deltas into smaller ones that compress even better.
 
 Using these adjusted values, if all the deltas are the same, the time range is stored using run-length encoding.
-If run-length encoding is not possible and all values are less than 1 << 60 - 1 (~36.5 yrs in nanosecond resolution), then the timestamps are encoded using simple8b encoding which is a 64bit word-aligned integer encoding.
-This encoding packs up multiple integers into a single 64bit word.
-If any value exceeds the maximum values, the deltas are stored uncompressed using 8 bytes each for the block.
+If run-length encoding is not possible and all values are less than (1 << 60) - 1 ([~18.3 years](https://www.wolframalpha.com/input/?i=(1+%3C%3C+60)+-+1+nanoseconds+to+years) at nanosecond resolution), then the timestamps are encoded using [simple8b encoding](https://github.com/jwilder/encoding/tree/master/simple8b).
```
Comment (jwilder) [inline content/influxdb/v1.0/concepts/storage_engine.md]: Should be 36.5 years.  18.3 is 1 << (60-1)

**Comment by beckettsean** [comment]
Thanks, @jwilder, both comments addressed.

Any other concerns or is this ready to merge? @pauldix any feedback?

**Comment by jwilder** [comment]
LGTM 👍

**Pull request merged**